sessions/backend-update-and-delete/products/pages/[id].js

```diff
@@ -0,0 +1,9 @@
+import Product from "../components/product";
+
+export default function ProductDetailsPage() {
+  return (
+    <>
```

Comment:
On the details page is no `h1` element. Maybe we can display the "Fish-Shop" heading on the details page as well. 🙂

Reply:
I see your point. However, The `Product` component returns an `<h1>Loading</h1>` while loading, and I am not sure whether we will have two `h1` at the same time (cannot really test it because the time while loading is to short to see what is displayed).

Besides that, this would mean to restructure the former challenges as well 🫢

Reply:
Ah ok, I understand 🤔